org.eclipse.jdt.ls.tests/projects/maven/salut5/proj/pom.xml

```diff
@@ -7,7 +7,7 @@
 		<plugins>
 			<plugin>
 				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.8.2</version>
```

Comment:
I think we can merge this assuming some of those small preference moves I mentioned can be done. Any idea why 3.8.0 is good but not 3.8.0 for the test ?

Reply:
There isn't maven-compiler-plugin 3.8.2 - https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-compiler-plugin
I'm not sure how it worked.